components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.publisher/src/main/java/org/wso2/carbon/apimgt/usage/publisher/dto/enums/EVENT_TYPE.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.wso2.carbon.apimgt.usage.publisher.dto.enums;
+
+/**
+ * ENUM to denote event types
+ */
+public enum EVENT_TYPE {
```

Comment:
Why do we have two enum classes having fault event types?

Reply:
EVENT_TYPE contains types of events. FAULT_EVENT_TYPE contains types of faulty events.

Reply:
It is kind of confusing to have fault events in both of those ENUMS. Shouldn't we just have two EVENT_TYPEs? 
1. success
2. fault

Reply:
But we use FAULT_EVENT_TYPE to keep all the faulty sub-event types. Maybe we can move RESOURCE_NOT_FOUND,
    API_NOT_FOUND and  METHOD_NOT_ALLOWED to FAULT_EVENT_TYPEs.

Reply:
yeah +1

Reply:
Fixed